cmd/cmd.go

```diff
@@ -151,6 +151,14 @@ func (m *Manager) Register(command Command) {
 		m.Commands = make(map[string]Command)
 	}
 	name := command.Info().Name
+	m.RegisterCommandByName(name, command)
+	alias := command.Info().Alias
+	if alias != "" {
+		m.RegisterCommandByName(alias, command)
+	}
```

Comment:
I'm not 100% sure but it could print the command info twice on help message - in different lines, though.

I'd take a better look on how we produce that help message and I'd show the comand + alias in same line - separeted by comma. For example:
```
app log, app logs                    Shows log entries for an application
```

Or maybe register the alias as a hidden command so it won't be generated on the help message even though it works fine when you fire on the command line.

Reply:
You are correct, with this changes, help message show twice `app log`

```sh
./bin/tsuru app
App is a program source code running on Tsuru

The following commands are available in the "app" topic:

[...]
  app log                    Shows log entries for an application
  app log                    Shows log entries for an application
[...]
```

I tried to change where is printed help message, but didn't find.

So I didn't append command alias inside `m.topicCommand`, with that change, only show once at help message:

```sh
./bin/tsuru app
App is a program source code running on Tsuru

The following commands are available in the "app" topic:

  [...]
  app list                   Lists all apps that you have access to
  app log                    Shows log entries for an application
  app metadata get           Retrieves metadata for an application
  [...]
```

What do you think @nettoclaudio ?

Reply:
My last comment was about when you call `tsuru app` to show all available commands.

But this other test, I only call `tsuru` and the real command and the alias was in the list, is that correct?!

```sh 
./bin/tsuru
Client version: dev.

Usage: tsuru command [args]

Available commands:
  [...]
  app list                   Lists all apps that you have access to
  app log                    Shows log entries for an application
  app logs                   Shows log entries for an application
  app metadata get           Retrieves metadata for an application
  [...]
```

Reply:
I'm OK on this approach.

To be honest, I don't like to maintain this code on our side anymore. There are a lot of better CLI modules (e.g. spf13/cobra, urfave/cli and so on) than ours. I'm in favor to use them and remove this bunch of code. I don't know if all Tsuru maintainers agree, just my opnion.